cmd/habitat/procs/manager.go

```diff
@@ -78,6 +78,29 @@ func (m *Manager) StartProcess(req *ctl.Request) (error, string) {
 	return nil, subName
 }
 
+// start arbitrary process
+func (m *Manager) StartArbitraryProcess(name string, cmdPath string, dataPath string, env []string, flags []string, args []string) (error, string) {
```

Comment:
Is this for starting a process but bypassing the app config and reverse proxy stuff (for IPFS I'm assuming)? I think it'd be good if we made an internal `startProcessInternal` call that basically does this stuff without all the config stuff, and have the og `StartProcess` function call the `startProcessInternal` function. That way we can keep the code for ensuring names are unique consistent since they are doing the same underlying thing

Reply:
this was never used in this PR, so i added a comment and I'm going to fix it in a follow up